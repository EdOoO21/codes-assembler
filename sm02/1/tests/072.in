601 217 512 -619 -296 431 114 269 -234
