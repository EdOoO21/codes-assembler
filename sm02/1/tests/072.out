-234
269
114
431
-296
-619
512
217
601
